src/engine/SCons/Tool/packaging/rpm.py

```diff
@@ -232,16 +230,23 @@ def build_specfile_header(spec):
         'X_RPM_BUILDROOT'     : 'BuildRoot: %s\n', }
 
     # fill in default values:
-    # Adding a BuildRequires renders the .rpm unbuildable under System, which
+    # Adding a BuildRequires renders the .rpm unbuildable under systems which
     # are not managed by rpm, since the database to resolve this dependency is
     # missing (take Gentoo as an example)
-#    if not s.has_key('x_rpm_BuildRequires'):
-#        s['x_rpm_BuildRequires'] = 'scons'
+    #if 'X_RPM_BUILDREQUIRES' not in spec:
+    #    spec['X_RPM_BUILDREQUIRES'] = 'scons'
 
     if 'X_RPM_BUILDROOT' not in spec:
         spec['X_RPM_BUILDROOT'] = '%{_tmppath}/%{name}-%{version}-%{release}'
 
     str = str + SimpleTagCompiler(optional_header_fields, mandatory=0).compile( spec )
+
+    # github #3164: if we don't turn off debug package generation
+    # the tests which build packages all fail.  This just slams
+    # a flag into the specfile header, could make it an X_RPM_NODEBUGPKGS
```

Comment:
Please go ahead and add this variable.  If our tests are having this issue, users will/may as well.